docker-compose/docker-compose-injistack/config/mimoto-issuers-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "issuers": [
+    {
+      "credential_issuer": "Mock",
+      "protocol": "OpenId4VCI",
+      "display": [
+        {
+          "name": "Farmers Credentials",
+          "logo": {
+            "url": "https://api.collab.mosip.net/inji/mosip-logo.png",
+            "alt_text": "mosip-logo"
+          },
+          "title": "Mock Identity",
+          "description": "Download Mock Identity Credential",
+          "language": "en"
+        }
+      ],
+      "client_id": "mpartner-mock-testing",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+      "token_endpoint": "http://localhost:8099/v1/mimoto/get-token/Mock",
+      "authorization_audience": "https://esignet-mock.dev1.mosip.net/v1/esignet/oauth/v2/token",
+      "proxy_token_endpoint": "https://esignet-mock.dev1.mosip.net/v1/esignet/oauth/v2/token",
+      "client_alias": "mpartner-mock-testing",
+      "qr_code_type": "EmbeddedVC",
+      "enabled": "true",
+      "wellknown_endpoint": "http://certify:8090/v1/certify/issuance/.well-known/openid-credential-issuer"
```

Comment:
this wellknown endpoint has to be localhost:8090, otherwise mobile won't be able to access it

Reply:
this setup is completely with injiweb, injiweb wont be able to access it on localhost